distro/pom.xml

```diff
@@ -75,9 +76,14 @@
         </dependency>
         <dependency>
             <groupId>org.bahmni.module</groupId>
-            <artifactId>bahmnicore-omod</artifactId>
+            <artifactId>bahmnicore-ts-omod</artifactId>
             <version>${bahmniCoreVersion}</version>
         </dependency>
+        <dependency>
+            <groupId>org.bahmni.module</groupId>
+            <artifactId>terminology-services-omod</artifactId>
```

Comment:
can we be more specific with the name? like fhir-ts-services-omod? or "snomed-ts-services-omod"?

Reply:
sure @angshu, will go with `fhir-ts-services-omod` name.